src/octoprint/plugins/softwareupdate/__init__.py

```diff
@@ -225,20 +226,20 @@ def get_settings_defaults(self):
 		return {
 			"checks": {
 				"octoprint": {
-					"type": "github_release",
-					"user": "foosel",
```

Comment:
You still need a "type" key, you can't just remove it like that

Reply:
But if it is here I can't change it with the hook, it will be overwritten by this type

Reply:
I will check again